src/elements/Quad.H

```diff
@@ -96,24 +118,14 @@ namespace impactx::elements
             amrex::ParticleReal & AMREX_RESTRICT py,
             amrex::ParticleReal & AMREX_RESTRICT pt,
             uint64_t & AMREX_RESTRICT idcpu,
-            RefPart const & refpart
+            [[maybe_unused]] RefPart const & refpart
```

Comment:
Idea: I would remove this fully from the interface to prevent that we can write inefficient code.

Reply:
Discussed: this will not work to be fully removed, since the `refpart` for elements that integrate through maps is non-const.